frontend/src/features/projects/spl/steps/SplStep.tsx

```diff
@@ -159,8 +161,8 @@ const SplStep = ({ formikRef }: IStepProps) => {
             formikRef,
             submitStatusCode,
             getStatusTransitionWorkflow(submitStatusCode),
-          ).then((project: IProject) => {
-            actions.setValues(project);
+          ).then((project: IApiProject) => {
+            actions.resetForm({ values: toFlatProject(project) });
```

Comment:
two problems fixed here.
1) we want to set the initial state to whatever we get from the api, so reset form does that for us.
2) the format of the data returned by the api needs to be transformed before we inject it into formik.